Sdk.zig

```diff
@@ -926,7 +926,8 @@ fn createLibCFile(sdk: *const Sdk, version: AndroidVersion, folder_name: []const
     try writer.writeAll("gcc_dir=\n");
 
     const step = sdk.b.addWriteFile(fname, contents.items);
-    return step.getFileSource(fname) orelse unreachable;
+    // return step.getFileSource(fname) orelse unreachable;
+    return step.addCopyFile(.{ .path = fname }, "");
```

Comment:
`addCopyFile` isn't the function you want here. We're already writing out the file  using `addWriteFile`, we just need to get a `FileSource` from it. According to the message in the deprecated `getFileSource()` function, we need to use `step.files.items[0].getPath()` here.